tdvt/samples/connector-tests/connectionbuilder.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<testCases>
+<!-- Driver Present -->
+<connection  installedDrivers64='PostgreSQL Unicode(x64)'  bitness='64' class='postgres_odbc' username='test' password='p' dbname='testv1' odbc-connect-string-extras='' port='8912' schema='' server='testServer' >DRIVER={PostgreSQL Unicode(x64)};DATABASE=testv1;SERVER=testServer;UID=test;PWD=p;PORT=8912;BOOLSASCHAR=0;LFCONVERSION=0;UseDeclareFetch=1;Fetch=2048</connection>
```

Comment:
Don't forget ssl mode: https://github.com/tableau/connector-plugin-sdk/blob/dev-2020.4/samples/plugins/postgres_odbc/connectionResolver.tdr

Also match default port from UI for consistency.

Reply:
Added sslmode='require' and set post to 5432 which is the default post for postgres_odbc in UI